### PR TITLE
Some small examples to start with in README.md

### PR DESCRIPTION
**Pull request by sebasguts**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,31 @@ To install this module in Julia, use
 ] add https://github.com/oscar-system/GAP.jl
 ```
 
+## Basic usage
+
+After
```
Comment (fingolfin) [inline README.md]:
```suggestion
After entering the following in Julia,
```
This way it is clear that we are talking about Julia code, and it applies both to interactive and non-interactive usage.

File: README.md [hunk 2]
```diff
@@ -15,6 +15,31 @@ To install this module in Julia, use
...
+```julia
+using GAP
+```
+one can call all GAP functions via prefixing them with `GAP.Globals.`:
```
Comment (fingolfin) [inline README.md]:
```suggestion
one may call any GAP function by prefixing its name with `GAP.Globals.`. For example:
```

File: README.md [hunk 3]
```diff
@@ -15,6 +15,31 @@ To install this module in Julia, use
...
+```julia
+julia> GAP.Globals.SymmetricGroup( 3 )
+GAP: SymmetricGroup( [ 1 .. 3 ] )
+```
+The Julia types `Int64` and `Bool` are automatically converted to GAP, several others can be converted using `GAP.julia_to_gap`:
```
Comment (fingolfin) [inline README.md]:
```suggestion
The Julia types `Int64` and `Bool` are automatically converted to GAP objects.
Several others types of objects can be converted using `GAP.julia_to_gap`:
```

File: README.md [hunk 4]
```diff
@@ -15,6 +15,31 @@ To install this module in Julia, use
...
+```julia
+julia> x = GAP.julia_to_gap([1,2,3])
+GAP: [ 1, 2, 3 ]
+```
+Converting back to Julia is done using `GAP.gap_to_julia`, adding the type of the resulting object
```
Comment (fingolfin) [inline README.md]:
```suggestion
Converting back to Julia is done using `GAP.gap_to_julia`. However, for this one needs to specify the desired type of the resulting object. For example, to convert the GAP list of integers we just defined back to Julia, we might do this:
```

**Comment by fingolfin** [comment]
(Of course I am not asking you to write that manual here -- the change request is purely for some phrasing in you PR).

**Comment by sebasguts** [comment]
I see. I applied your suggests :)

File: README.md [hunk 5]
```diff
@@ -15,6 +15,32 @@ To install this module in Julia, use
...
+julia> GAP.Globals.SymmetricGroup( 3 )
+GAP: SymmetricGroup( [ 1 .. 3 ] )
+```
+The Julia types `Int64` and `Bool` are automatically converted to GAP objects.
+Several others types of objects can be converted using `GAP.julia_to_gap`:
+```julia
+julia> x = GAP.julia_to_gap([1,2,3])
+GAP: [ 1, 2, 3 ]
+```
+Converting back to Julia is done using `GAP.gap_to_julia`. However, for this one needs to specify the desired type of the resulting object. For example, to convert the GAP list of integers we just defined back to Julia, we might do this:
```
Comment (fingolfin) [inline README.md]: Why is this one long line, while lines 29/30 is wrapped?

File: README.md [hunk 6]
```diff
@@ -15,6 +15,32 @@ To install this module in Julia, use
...
 
+## Basic usage
+
+After entering the following in Julia,
+```julia
+using GAP
+```
+one may call any GAP function by prefixing its name with `GAP.Globals.`. For example:
+```julia
+julia> GAP.Globals.SymmetricGroup( 3 )
```
Comment (fingolfin) [inline README.md]: I am not going to request a change for this, but I note that once more, this is formatted differently in terms of whitespace than the other example inputs :-).

Comment (sebasguts) [reply README.md]: I think because you made this line longer because of your suggestions. Anyway, fixed it.

File: README.md (hunk 5 above)
Comment (sebasguts) [reply README.md]: I think because you made this line longer because of your suggestions ;). Anyway, fixed it.

File: README.md (hunk 6 above)
Comment (sebasguts) [reply README.md]: Still fixed it.

**Pull request merged**